platform/translation_keys/batch_operations.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: batch_operations
+title: Batch Operations
+sidebar_label: Batch Operations
+---
+
+Batch operations allow you to perform one operation on multiple keys at once.
+This is useful for performing operations on a many keys, such as deleting all keys in a namespace or machine translating all keys in a project.
+
+## Selecting keys
+
+First, select the keys you want to perform the batch operation on. You can do this by **selecting the checkbox next to the keys** you want to select.
+
+You can also **select all keys** by clicking the "Select all" checkbox next to the batch operation dropdown list. When using the "Select all" checkbox after performing a search or applying filters, it will only select all the keys that match the search query or filter conditions.

```

Comment:
We should mention that the selection popover is visible only when some keys are selected.